webpack.config.js

```diff
@@ -24,8 +24,35 @@ const path = require('path');
 module.exports = {
   entry: './src/index.js',
   devtool: 'inline-source-map',
-  target: 'node',
+  target: 'web',
   mode: 'development',
+  externals: {
```

Comment:
Is this anything to do with the `not found` errors when metro was bundling things together?